index.js

```diff
@@ -91,7 +92,7 @@ exports = module.exports = function (app, settings) {
 
 
   app.context.render = function *(view, _context) {
-    var context = {};
+    var context = settings.context || {};
```

Comment:
1. `settings.context` will changed every time when calling `ctx.render`
2. rename `context` to `state` to keep the same with `koa`